City-of-Helsinki/helsinki-paatokset
# PP-260: PP-260: styled the organization above in paattajat view

Ticket: https://helsinkisolutionoffice.atlassian.net/browse/PP-260

Styled the sub title in päättäjät-page.

**How to test**

Go to https://helsinki-paatokset.docker.so/fi/paattajat (if not created, make a page with the path)

Check the styles in luottamushenkilöpäättäjä and viranhaltijat. The sub title should now be smaller and in gray color
